repo_tests/ansible_manifest_resolution_test.py

```
@@ -0,0 +1,280 @@
+sees the live tree cannot be shown to REJECT anything, and the repo standard is
+a contrast pair per detector. `ansible_manifest_resolution_contrast_test.py`
+supplies those pairs against synthetic tables; this file supplies the live half.
+
+Each comparison also states the REACH it needs before it compares. A floor bound
+to the number of findings passes in silence when the parser breaks, and this
+whole issue family is about a guard that could not see what it claimed to read.
+"""
+
+from __future__ import annotations
+
+from typing import Callable
```

Ruff (0.16.3)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)